src/tests/multihost/alltests/conftest.py

```diff
@@ -699,6 +699,30 @@ def removeproxyldap():
     request.addfinalizer(removeproxyldap)
 
 
+@pytest.fixture(scope='function')
+def proxy_sleep(session_multihost, request):
+    """ Create sssd proxy pam ldap config file """
+    execute_cmd(session_multihost, "echo 'auth required pam_exec.so "
+                                   "/usr/bin/sleep 100' > "
+                                   "/etc/pam.d/proxy_sleep")
+    execute_cmd(session_multihost, "echo 'password required pam_exec.so "
+                                   "/usr/bin/sleep 100' "
+                                   ">> /etc/pam.d/proxy_sleep")
+    execute_cmd(session_multihost, "echo 'account required pam_exec.so "
+                                   "/usr/bin/sleep 100' >> "
+                                   "/etc/pam.d/proxy_sleep")
+    execute_cmd(session_multihost, "echo 'session required pam_exec.so "
+                                   "/usr/bin/sleep 100' >> "
+                                   "/etc/pam.d/proxy_sleep")
+
+    def removeproxyldap():
+        """ Remove sssd proxy pam ldap config file """
+        remote = '/etc/pam.d/proxy_sleep'
+        cmd = 'rm -f %s' % remote
+        session_multihost.client[0].run_command(cmd)
+    request.addfinalizer(removeproxyldap)
+
```

Comment:
Can we move this fixture in the same file as the test code? It is specific for 1 test only, so no need of adding it to conftest.py

Reply:
Done